src/main/java/Task.java

```diff
@@ -0,0 +1,66 @@
+public class Task {
+
+    // Keep track of tasks
+    private static Task[] tasks = new Task[100];
+    private static int taskNumber = 0;
+
+    // Object specific variables
+    public String taskString;
+    public boolean taskDone = false;
```

Comment:
Should the boolean name here be more intuitive like in the Java coding standard?